packages/Webkul/API/Http/Controllers/Shop/CartController.php

```diff
@@ -69,7 +72,7 @@ public function __construct(
     /**
      * Get customer cart
      *
-     * @return \Illuminate\Http\Response
+     * @return JsonResponse
```

Comment:
annotations should have fully qualified class names even if class is loaded in use section

Reply:
True, done.